cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/profiles_populator.py

```diff
@@ -0,0 +1,428 @@
+import argparse
```

Comment:
I will add a README section once the final shape of the tool is determined

Reply:
I'd prefer if you wrote the README first and let me review it as a design proposal. It would surely make code reviews much easier for me and it may save you work too.

Reply:
OK